versions.tf

```
@@ -20,7 +20,7 @@ terraform {
 
     google = {
       source  = "hashicorp/google"
-      version = ">= 3.53, < 5.0"
+      version = ">= 4.1.0, < 5.0"
```

Sorry typo, it was in 4.10 https://github.com/hashicorp/terraform-provider-google-beta/blob/main/CHANGELOG.md#4100-february-7-2022
```suggestion
      version = ">= 4.10.0, < 5.0"
```